frontend/src/serie.html

```diff
@@ -0,0 +1,69 @@
+    size: 1
+    alias: serie
+    addAllPagesToCollections: true
+permalink: "series/{{ serie.title | slugify }}/"
+layout: base
+tags: series
+eleventyComputed:
+  title: "{{ serie.title }} Lecture Series"
+jsFile: common-player.js
+---
+<article class="mx-auto mt-10 max-w-screen-md px-2 sm:px-4">
+  <!-- Title -->
+  <h1 class="flex items-center space-x-1 rounded-lg bg-neutral-600 p-6 text-xl font-bold text-white">
+    <!-- https://icon-sets.iconify.design/cil/featured-playlist/ -->
+    <svg xmlns="http://www.w3.org/2000/svg" width="1em" height="1em" viewBox="0 0 512 512">
+      <path fill="currentColor" d="M464 72H48a32.036 32.036 0 0 0-32 32v304a32.036 32.036 0 0 0 32 32h416a32.036 32.036 0 0 0 32-32V104a32.036 32.036 0 0 0-32-32m0 336H48V104h416l.02 304Z"/><path fill="currentColor" d="M232 184h184v32H232zm-56 72h240v32H176zm0 72h240v32H176zM88.923 144v128.923l99.172-69.42z"/>
+    </svg>
+    <span>{{ serie.title }}</span>
+  </h1>
+  <ul class="mt-2">
+    {%- for audio in serie.audios %}
+      <li
+        x-data="audioItem({{ audio | json | escape }})"
+        class="mb-2 flex rounded-md border border-neutral-500 pb-1"
+      >
+        {% render 'partials/play-button.html' -%}
+        <div class="flex flex-grow flex-col">
+          <div class="flex items-baseline justify-between">
+            <!-- Title -->
+            <div class="text-sm font-medium">{{ audio.audios_id.title }}</div>
+            <!-- Duration -->
+            <div class="inline-flex content-end items-center pr-2 text-xs font-medium">
+              {% assign audioDuration = audio.audios_id.duration %}
+              {% render 'partials/duration.html' with audioDuration as duration -%}
+            </div>
+          </div>
+          <!-- Second line -->
+          <div class="mt-1 flex items-start justify-between space-x-2 pr-2 text-xs">
+            <div class="flex flex-grow flex-wrap items-start justify-between">
+              <!-- Tags -->
+              <div class="mb-1 flex flex-wrap content-between items-baseline [&>*]:mb-1 [&>*]:mr-1 [&>*]:border-r [&>*]:border-gray-300 [&>*]:pr-1 last:[&>*]:border-0">
+                <div title="Unique file identifier">#{{ audio.audios_id.id }}</div>
+                {% render 'partials/audio-tags.html', audio: audio.audios_id -%}
+              </div>
+              <div class="mb-1 flex items-center space-x-1">
+                <!-- Colored badges -->
+                {% assign audioPercentage = audio.audios_id.percentage %}
+                {% render 'partials/percentage.html' with audioPercentage as percentage -%}
+                {% assign audiosoundQualityRating = audio.audios_id.soundQualityRating %}
+                {% render 'partials/sound-quality.html' with audiosoundQualityRating as soundQualityRating -%}
+              </div>
+            </div>
+            <!-- Actions -->
+            {% render 'partials/audio-actions.html', fileId: audio.audios_id.id, title: audio.audios_id.title -%}
```

Comment:
After we moved all the HTML into a partial, there is no need for the comment above it. It's obvious what follows without a comment.